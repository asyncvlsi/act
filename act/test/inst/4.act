defproc foo(bool a, b) ;

bool p, s;

foo f[3](p,s);
