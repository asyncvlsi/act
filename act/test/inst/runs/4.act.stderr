In expanding ::<Global>
 id: f.a
FATAL: Missing array dereference for ``f''.
